<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="file_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>460</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>파일명</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="matplotlibWidget" name="widget_graph" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_graph">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>190</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_log">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QListView" name="log_view">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>190</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="store_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>store</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autosave_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>auto save</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>42</width>
       <height>31</height>
      </rect>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="widget_image">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidgetFile</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
